pkg/tracing/basictracing.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2017 Google Inc.
```

Comment:
Derp, done. This resulted in me updating the stdout recorder to take an io.Writer instead.